src/main/java/uk/gov/register/db/EntryQueryDAO.java

```diff
@@ -47,7 +47,7 @@
     @FetchSize(262144)
     ResultIterator<Entry> getIterator();
 
-    @SqlQuery("SELECT * FROM entry WHERE entry_number between :startEntryNo and :endEntryNo ORDER BY entry_number")
+    @SqlQuery("SELECT * FROM entry WHERE entry_number > :startEntryNo and entry_number <= :endEntryNo ORDER BY entry_number")
```

Comment:
If we call /download-rsf/1/2, we only want to get entry #2, which this change ensures.